### DISCUSSION TITLE
Enchant Cloak - Greater Fire/Nature Resistance and the TBC material grind.

### DISCUSSION THREAD

**Discussion by Zelfie** [chromiecraft/chromiecraft#3888] [Suggestions]
I feel the current choices regarding this particular material grind go against the intended design, and are a huge disrespect to players, and their time.

First of all lets take a look at the Classic requirements for both these enchants:

Enchant Cloak - Greater Fire Resistance
3x Nexus Crystal, 8x Large Brilliant Shard, 4x Essence of Fire

Enchant Cloak - Greater Nature Resistance 
2x Nexus Crystal, 8x Large Brilliant Shard, 4x Living Essence

Some epic disenchants, some blue disenchants. Fair.

TBC changed this as it was still a viable enchant, so changed the mats to suit the current expansion to stop people having to farm old content for these enchants, and instead it became:
2/3x Small Prismatic Shard, 3x Greater Planar Essences, 4x Fire/Living Essence.

So with these unavailable, how do we solve this problem?
Change the recipe? That probably requires a custom client or something?
Give us the Nexus Crystal Transformation enchanting ability early? That only solves the TBC shard
Add a material trader? 

So you went with add a material trader. Cool.
It balances out almost the same.
1x Nexus Crystal for 1x TBC shard. Fair
9x Large Brilliant Shards for the 3x Greater Planar Essences. Almost, we pay 1LBS tax basically. Okay.

Then, out of nowhere. With no announcement.
TBC shard trade removed. 

So let's go step by step why this sucks.

There's very few things you can do with Nexus Crystals : Sageblade. AQ Enchants. Fire/Nature Cloak enchants.
You've removed the cloak enchants from that pool, without giving us warning, which is insulting for the guilds that have been saving up some Nexus Crystals for the big nature res farm that's about to begin.

ThEy ArE iNtEnTiOnAlLy ReMoVeD aS tRaDe OpTiOn On RoBoTo
Yeah, without warning. It is insulting to the playerbase to move the goalposts without warning, when giving us the "Nexus Transformation" enchanting spell early would be a viable move instead.

You've changed the physical requirements of the enchants.
In a guild setting, you've changed what is required to acquire them. In normal raid guild play you likely won't have an abundance of elemental essences, they become the materials to farm for things like this. Epic crystals and Blue shards come from the raids and adventures, and they make these enchants.
Instead this dynamic has changed to "farm ZG/AQ20". That's it. That's how you get these. No other way. Just "some" items that allegedly disenchant into what you need. And they aren't even 100% chance! If we're going off Wotlk Wowhead there's 11%-50% for those TBC shards to disenchant from TWO BOSSES in ZG. 
-EDIT- (This argument was based on WOTLK Wowhead information, where the tbc shards are 11-50% from different blues, WotlkDB states them as being 99.5, I'm unsure what the azerothcore drop rate is) But its still not 100%.
How miserable has the grind become for these enchants?
It goes savagely against player interest to gate this stuff behind ZG and allegedly AQ20. 

Server population.
With the recent hemorrhage of players from a certain guild around certain drama and summer having it's usual effect on time-played. Many people have left. Maybe overall population hasn't changed too much, hopefully the promise of ZG and AQ have brought some new people in, but those people will want gear from the catchup raids, which destroys the amazing 11% chance I've got of disenchanting a TBC blue off that item if a newb needs it. Adding to the long, drawnout grind it would require to get everyone a 15 nature res cloak enchant.

CONSPIRACY THEORY TIME
I assume whoever reported this "oh look you can remove that trade now", reported that after their guild had traded in a few hundred Nexus Crystals, and conveniently for them they don't need to collect any more.
If you're removing all the TBC mats at the start of TBC, then it's not like we gain much long-term out of Robotos trade being restored. We just have something to do with our Nexus Crystals, and aren't being forced to run AQ20 and ZG every 3 days for a CHANCE at TBC mats due to questionable decision making.

**Comment by 55Honey** [comment]
Hey Zelfie, thanks for bringing it up.
We'll look into this and get back to you once it was reviewed.

**Comment by heyitsbench** [comment]
> ThEy ArE iNtEnTiOnAlLy ReMoVeD aS tRaDe OpTiOn On RoBoTo

There's no reason to write it this way.

**Comment by 55Honey** [comment]
Hi everyone,

We've looked over the situation and decided to give the turn-in option for Nexus Crystals a fabulous return.
It's not *just* going to be added back as it was before. Instead, Roboto got a total overhaul and will allow turn-ins of 1, 5, 10, or 20 items at once.

![image](https://user-images.githubusercontent.com/71938210/183460010-168ec373-1e70-4e7f-acae-789a9313d10a.png)


This change is subject to being live when the epic PvP items are released.

Thanks for bringing it up and I hope you will enjoy the new features.

**Reply by Zelfie** [reply]
Thank you Honey, and I'm loving the 1/5/10/20, oh what a lovely addition.
Thank you for taking the grind off and giving our Nexus Crystals some use and value.
Keep up the good work.
